Source/RxSwift/Observable+Moya.swift

```diff
@@ -37,9 +37,9 @@ extension ObservableType where E == Response {
     }
 
     /// Maps data received from the signal into a JSON object. If the conversion fails, the signal errors.
-    public func mapJSON() -> Observable<AnyObject> {
+    public func mapJSON(failsOnEmptyData failsOnEmptyData: Bool = true) -> Observable<AnyObject> {
```

Comment:
Well done with including the call-site parameter name, I would've forgotten that.